qiskit_ibm_provider/accounts/management.py

```diff
@@ -95,10 +95,7 @@ def _matching_default(account_name: str) -> bool:
 
         # load all accounts
         all_accounts = map(
-            lambda kv: (
-                kv[0],
-                Account.from_saved_format(kv[1]),
-            ),
+            lambda kv: (kv[0], Account.from_saved_format(kv[1]),),
```

Comment:
@rathishcholarajan there are a lot of unrelated formatting changes that also cause the CI to fail. Can you rerun black and commit the changes? Makes it easier to focus on the relevant changes only :). Happy to take a deeper look afterwards.

Reply:
Fixed!